src/core/drive/page_snapshot.js

```diff
@@ -83,6 +83,10 @@ export class PageSnapshot extends Snapshot {
     return this.getSetting("refresh-scroll")
   }
 
+  get visitScroll() {
+    return this.getSetting("visit-scroll")
+  }
```

Comment:
The current `<meta name="turbo-scroll">` is meant to parallel the concept of a "refresh". 

Originally, this PR proposed `<meta name="turbo-scroll">`. An alternative couldbe `<meta name="turbo-render-scroll">`. Are either of those preferable?